tests/test_browser.py

```diff
@@ -3911,6 +3911,19 @@ def test_pthread_tls_main(self):
   def test_pthread_lsan(self, name, args=[]):
     self.btest(path_from_root('tests', 'pthread', name + '.cpp'), expected='1', args=['-fsanitize=leak', '-s', 'TOTAL_MEMORY=256MB', '-s', 'USE_PTHREADS', '-s', 'PROXY_TO_PTHREAD', '-std=c++11', '--pre-js', path_from_root('tests', 'pthread', name + '.js')] + args)
 
+  @parameterized({
+    'leak': ['test_pthread_lsan_leak', ['-g4']],
+    'no_leak': ['test_pthread_lsan_no_leak'],
```

Comment:
`lsan`->`asan`?

Reply:
No. We are reusing the test_pthread_lsan* for ASan. I'll add a comment to say this is not an error.